merkledag/merkledag.go

```diff
@@ -371,27 +410,27 @@ func EnumerateChildrenAsync(ctx context.Context, ds DAGService, root *Node, set
 func fetchNodes(ctx context.Context, ds DAGService, in <-chan []key.Key, out chan<- *Node, errs chan<- error) {
 	defer close(out)
```

Comment:
I see -- that makes sense. Is there any possibility of missing a node if you 1. send the node, then 2. return, thus closing the node channel?